pkg/cli/sql_util_test.go

```diff
@@ -98,12 +98,10 @@ func TestRunQuery(t *testing.T) {
 	conn := makeSQLConn(url.String())
 	defer conn.Close()
 
-	// Use a buffer as the io.Writer.
-	var b bytes.Buffer
-
-	cliCtx.tableDisplayFormat = tableDisplayPretty
+	setCLIDefaultsForTests()
 
 	// Non-query statement.
+	var b bytes.Buffer
```

Comment:
This gets used for a few other statements, so I'd maybe leave it above the comment.

Reply:
Aye, done.